### DISCUSSION TITLE
Where can I find the source code?

### DISCUSSION THREAD

**Discussion by kmaki565** [Purfview/whisper-standalone-win#370] [Q&A] [answered]
Hello @Purfview , thank you for this amazing repo! This is the most comprehensive & easy-to-use tool I've ever found among all Whisper variants.

Do you publish the source code or did I just overlook? I'd love to learn from it and contribute the project if possible.

**Comment by Purfview** [comment, marked as answer]
Here is not published any code.

There you can contribute:
https://github.com/SYSTRAN/faster-whisper
https://github.com/openai/whisper
